src/LexicographicHelpers.dfy

```diff
@@ -0,0 +1,243 @@
+// RUN: %dafny /compile:0 "%s"
+
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT 
+*******************************************************************************/
+
+include "Relations.dfy"
+include "Lexicographics.dfy"
+include "UInt.dfy"
+
+module LexicographicHelpers{
+    export 
+        provides LexIsReflexive, LexIsAntisymmetric, LexIsTransitive, LexIsTotal, UInt8LessIsTrichotomousTransitive
+        provides ThereIsAMinimum, MinimumIsUnique, FindMinimum
+        provides SetToOrderedSequence, IsMinimum
+        provides UInt, Relations, Lexicographics
+        provides  LessOrEqual, LessOrEqualAux
+
+    import opened Relations
+    import opened Lexicographics
```

Comment:
Why do you need Lexicographics here? I thought Lexicographic sequence bytes would to the contrary inherit from Lexicographic sequences that you are describing there.

Reply:
it was creating cyclic dependencies with imports, so I need to sort it out.